src/crimson/os/seastore/onode_manager/staged-fltree/fltree_onode_manager.h

```diff
@@ -47,6 +47,17 @@ struct FLTreeOnode final : Onode, Value {
       Value(std::forward<T>(args)...) {}
 
   struct Recorder : public ValueDeltaRecorder {
+    enum class delta_op_t : uint8_t {
+      UPDATE_ONODE_SIZE,
+      UPDATE_OMAP_ROOT,
+      UPDATE_XATTR_ROOT,
+      UPDATE_OBJECT_DATA,
+      UPDATE_OBJECT_INFO,
+      UPDATE_SNAPSET,
+      CLEAR_OBJECT_INFO,
+      CLEAR_SNAPSET,
+      UPDATE_ALL
```

Comment:
Looks the only user of UPDATE_ALL is at the create time, which can be replaced by a more specfic and cost-effective delta type (probably CREATE_DEFAULT) which doesn't need to encode the entire value as payload, right?

Reply:
I think we need to encode the entire value, because we need to make sure the hole value field is all zero instead of unexpected value on create, am I right?

Reply:
I think making sure the whole value field is all zero doesn't mean to encode the entire value into the delta payload. CREATE_DEFAULT can be used to determine that the whole value should be default initialized without needing to encode the actual payload, right?

Reply:
Ah, yes.

Reply:
Done.